packages/nextjs/app/builders/0x74370B567f5c65bef0428B9c78df5C691B632Cf7/components/SocialLinks.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Github, Link, Linkedin, Twitter } from "lucide-react";
+
+const SocialLinks = () => {
+  const socialIcons = [
+    { name: "Twitter", url: "https://twitter.com/armolas_06", icon: Twitter },
+    { name: "GitHub", url: "https://github.com/armolas", icon: Github },
+    { name: "LinkedIn", url: "https://linkedin.com/in/arowolomuritadhor", icon: Linkedin },
+    { name: "Website", url: "https://armolasportfolio.netlify.app", icon: Link },
+  ].filter(social => social.url);
```

Comment:
There's no need for this `filter` since all social links have an `url`.
And since this is a constant, it makes sense to place it outside of the component. So it's created only once at module initialization, rather than every render.